IFC4x3/Constants/v/VEGETATION_039IZnw4P17vEJksQmuxAR.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="VEGETATION_039IZnw4P17vEJksQmuxAR" Name="VEGETATION" UniqueId="032528f1-e846-411f-9393-bb66b0e3b29b">
+	<Documentation>Plant life or plant cover (as of an area). For example trees, shrubs, herbs, grasses, ferns, and mosses.</Documentation>
+</DocConstant>
+
```

Comment:
I think that IfcPlant was primarily aimed towards something man-made and not existing vegetation. Is this distinction clear?

Reply:
Definitely not. But this is what was submitted to ISO, so we should complain to ISO I guess.